### ISSUE TITLE
Analytics namespace missing and other empty directories

### ISSUE THREAD

**Issue by JanFromNaomiAnimation** [readyplayerme/rpm-unity-sdk-core#41]
**Describe the bug**
Analytics namespace doesnt exists and Unity ends up with error:
Library/PackageCache/com.readyplayerme.core@39d3c866dc/Editor/UI/Components/Footer.cs(3,26): error CS0234: The type or namespace name 'Analytics' does not exist in the namespace 'ReadyPlayerMe.Core' (are you missing an assembly reference?)

**To Reproduce**
1. Go to Unity
2. Open package manager
3. Add RPM core package from git url: https://github.com/readyplayerme/rpm-unity-sdk-core.git
4. Ready player me core ver. 0.1.1 installed
5. See error

**Expected behavior**
All necessary files in package

**Screenshots**
![image](https://user-images.githubusercontent.com/124244732/216268021-5f5ec058-859a-45ba-aaa3-9ccf2efaa557.png)

**Desktop (please complete the following information):**
 - Platform: Unity editor 2021.3.15f1, macOS Ventura 13.2

**Comment by rYuuk** [comment]
Unfortunately I am unable to reproduce it. Does you project have old RPM SDK?
![image](https://user-images.githubusercontent.com/1121080/216279815-0f755e0c-db41-4f25-a092-5d66a29a5e3b.png)

**Comment by JanFromNaomiAnimation** [comment]
At first we tried it without removing old SDK (as it was located in Plugins directory instead of packages and we were avare of duplicity) and when Analytics directory was empty for first time we tried to remove all RPM related files/assets based on: https://wolf3d.notion.site/Unity-SDK-2-0-Migration-Guide-32ca364853f44b35ac527bf61fa100ed#c4efb14911f94e0cafd43fcf83d0e324

Base on your question above we tried to use your package in completely new Unity URP project after that and in this scenario is is working like a charm.

What moves us forward was to remove project from Unity Hub, completely remove our project repository from local storage (even if we didnt have any unstaged files) and pull it again. Now we are in state, where all RPM files are in project and old API updated. 

Maybe it was not good move to try to install it without removing old SDK in our first step ¯\_(ツ)_/¯
Sorry in any inconveniences and thnx for your time.

Regards Jan
